spec/factories/tasks.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :task do
+    user
+    sequence(:project){"Project #"}
+    sequence(:title) {"Task #" }
```

Comment:
To use a sequence you should give a block with 1 argument - the number from the sequence
```ruby
sequence(:title) { |n| "Task ##{n}" } # will produce "Task #1", "Task #2" and so on.
```